src/model.js

```diff
@@ -326,6 +330,8 @@ fakeModel.prototype.findOne = function (obj) {
 	var self = this;
 	
 	return this.$query({
+		query: obj,
+		method: "findOne",
```

Comment:
Haha alright. Probably not necessary but definitely would be good to have at some point.